### PR TITLE
fix: Sync constants imports

### PR DESCRIPTION
**Pull request by pxrl**
Export directly from src/utils/index.ts so that we don't end up with stale references to contracts-v2.

### CODE REVIEW

File: src/utils/AddressUtils.ts
Comment (pxrl) [inline src/utils/AddressUtils.ts:1]: For files inside `src/utils` I have retained importing directoy from `constants-v2` because we don't want circular dependencies by importing from `src/utils/index.ts`. I tried to import and re-export from `src/utils/AddressUtils.ts` but ended up in tsc purgatory and ran out of patience. If anyone has any tips I'm happy to update it.

File: test/AdapterManager.SendTokensCrossChain.ts
Comment (pxrl) [inline test/AdapterManager.SendTokensCrossChain.ts:1]: This file contains a few related driveby fixes that I idenfitied whilst here.

File: src/utils/AddressUtils.ts
Comment (nicholaspai) [reply src/utils/AddressUtils.ts:1]: sounds good

File: src/clients/bridges/op-stack/optimism/OptimismAdapter.ts [hunk 1]
```diff
@@ -1,9 +1,7 @@
-import { winston } from "../../../../utils";
+import { winston, TOKEN_SYMBOLS_MAP } from "../../../../utils";
 import { SpokePoolClient } from "../../..";
```
Comment (james-a-morris) [inline src/clients/bridges/op-stack/optimism/OptimismAdapter.ts:2]: Completely out of scope, but this relative import is nuts. It could easily lead to circular dependencies because we're importing from an ancestor `index.ts` file.

File: src/utils/index.ts [hunk 2]
```diff
@@ -31,6 +31,8 @@ export { config } from "dotenv";
 export { replaceAddressCase } from "@uma/common";
 export { Logger } from "@uma/financial-templates-lib";
 
+export { CHAIN_IDs, TOKEN_SYMBOLS_MAP } from "@across-protocol/constants-v2";
```
Comment (james-a-morris) [inline src/utils/index.ts:34]: Thoughts on placing this high on the import chain in a separate file called `constants.ts`?

Reasoning: we already have this schema in `./utils/SDKUtils.ts` that exports. It would be consistent to do the same for the constants-v2 repo.

File: test/AdapterManager.SendTokensCrossChain.ts [hunk 3]
```diff
@@ -48,12 +47,12 @@ let l1BaseBridge: FakeContract;
...
-  weth: "0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2",
-  dai: "0x6B175474E89094C44Da98b954EedeAC495271d0F",
-  wbtc: "0x2260FAC5E5542a773Aa44fBCfeDf7C193bc2C599",
-  snx: "0xC011a73ee8576Fb46F5E1c5751cA3B9Fe0af2a6F",
-};
+  usdc: TOKEN_SYMBOLS_MAP.USDC.addresses[CHAIN_IDs.MAINNET],
+  weth: TOKEN_SYMBOLS_MAP.WETH.addresses[CHAIN_IDs.MAINNET],
+  dai: TOKEN_SYMBOLS_MAP.DAI.addresses[CHAIN_IDs.MAINNET],
+  wbtc: TOKEN_SYMBOLS_MAP.WBTC.addresses[CHAIN_IDs.MAINNET],
+  snx: TOKEN_SYMBOLS_MAP.SNX.addresses[CHAIN_IDs.MAINNET],
```
Comment (james-a-morris) [inline test/AdapterManager.SendTokensCrossChain.ts:54]: Nit: I'm not a huge fan of the reuse of `TOKEN_SYMBOLS_MAP[{SYMBOL}].addresses[CHAIN_IDs.MAINNET]` but nothing actionable.

**Review by james-a-morris — COMMENTED** [review]
This looks like a 1:1 refactoring. Correct me if I'm wrong, but it doesn't look like any business logic has been changed?

Only question: is there a reason we're not branched against master?

File: src/clients/bridges/op-stack/optimism/OptimismAdapter.ts (hunk 1 above)
Comment (james-a-morris) [reply src/clients/bridges/op-stack/optimism/OptimismAdapter.ts:2]: This may be worthy of adding to our tech debt cycle @nicholaspai

File: test/AdapterManager.SendTokensCrossChain.ts (hunk 3 above)
Comment (pxrl) [reply test/AdapterManager.SendTokensCrossChain.ts:54]: Yeah. I also think it could be done more efficiently, but didn't want to spend the time on it because it'd involved changing even more of this code. It works OK, but I thought it was preferable to avoid replicating official addresses, which is why I subbed this in.

**Comment by pxrl** [comment]
> This looks like a 1:1 refactoring. Correct me if I'm wrong, but it doesn't look like any business logic has been changed?

Correct - 1:1, no change in business logic.

> Only question: is there a reason we're not branched against master?

It depends on another PR that I have open, but I didn't want to pollute that with this plumbing.

File: src/utils/AddressUtils.ts
Comment (mrice32) [reply src/utils/AddressUtils.ts:1]: +1

**Review by mrice32 — APPROVED** [review]
LGTM!

File: src/clients/bridges/op-stack/optimism/OptimismAdapter.ts (hunk 1 above)
Comment (pxrl) [reply src/clients/bridges/op-stack/optimism/OptimismAdapter.ts:2]: I added a Linear ticket for it :+1:

File: src/utils/index.ts (hunk 2 above)
Comment (pxrl) [reply src/utils/index.ts:34]: I added a Linear ticket for it, bundled with the point above :+1:
